synapse/tests/test_model_base.py

```diff
@@ -77,6 +77,10 @@ class BaseTest(s_t_utils.SynTest):
                 self.eq(node.get('data'), ('some', 'data', 'here'))
                 self.eq(node.get('name'), 'Magic Pony')
 
+                # Raise on non-json-safe values
+                props['data'] = {(1, 2): 'foo'}
+                await self.asyncraises(s_exc.MustBeJsonSafe, snap.addNode('graph:event', iden, props=props))
```

Comment:
Do we have an explicit failure for bytes as well?

Reply:
ok